techdocs/docs/using-the-package/protecting-a-route.md

```diff
@@ -55,3 +62,6 @@ app.use("/vote", protectedRoute(['Member', 'Verified'], { requireAllRoles: false
 
 !!! tip "Tip"
     You can also check if the user has a role within the controller function by using the `hasRoles` function on the `SSOUser` object of `req.user`.
```

Comment:
note that hasroles and req.user are showing up as links that are broken on this page